src/libinjection_xss.c

```diff
@@ -135,6 +135,314 @@ static int html_decode_char_at(const char* src, size_t len, size_t* consumed)
     }
 }
 
+/*
```

Comment:
Adding the source for these in the code as described in the PR description may be useful?

Reply:
Added pointer to github webkit source.